src/core/server/saved_objects/service/lib/filter_utils.test.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import { fromKueryExpression } from '@kbn/es-query';
+import { esKuery } from '../../../../../plugins/data/server';
```

Comment:
I agree with @rudolf's previous comment here. As unfortunate it is to see this that far in the realisation process, having `core` depending on plugins is something we really needs to avoid. We would really like this to be re-evaluated.

Reply:
@pgayvallet @rudolf, doesn't this mean that it should be owned and exposed from core?

Reply:
This seems to be rudolf's second proposition. 

However, when I read https://github.com/elastic/kibana/issues/42885, One of the reason to move KQL to `data` plugin was
> This will help solve some problems of shared type interfaces (specifically Field and IndexPattern)

Don't have much context here, but would moving KQL into core instead of data plugin still adresse this point?

Second question will be regarding core owners yes. I guess moving it to core would mean platform team becomes the actual owner. Not sure how realistic this is and how everyone feels about it.

CC @joshdover @rudolf

Reply:
This has been discussed on slack between @rudolf and @lukeelmers. Summary is:

As there is nothing code breaking for core in the PR, and as the PR is blocking things than needs to ship for 7.6, it can be merged as is, and solutions about the code sharing will be addressed later